modules/op-release-notes-1-18.adoc

```diff
@@ -0,0 +1,373 @@
+        bundleresolver-config:
+          data:
+            fetch-timeout: 1m # <2>
+# ...
+----
+<1> Specify a global maximum timeout for resolution requests. The default value is `1m`.
+<2> Specify a timeout that is used for bundle resolution requests.
+
+
+[id="Operator-new-features-1-18_{context}"]
+=== Operator
+
+* With this release, the {pipelines-shortname} now support community tasks.
++
+The following community tasks are installed by default in the `openshift-pipelines` namespace:
++
+** `argocd-task-sync-and-wait`
+** `git-cli`
+** `helm-upgrade-from-repo`
+** `helm-upgrade-from-source`
+** `jib-maven`
+** `kubeconfig-creator`
+** `pull-request`
+** `trigger-jenkins-job`
+
+* With this release, {pipelines-shortname} supports the deployment of new containers through the `TektonConfig` CR.
++
+.Example of deploying a new `kube-rbac-proxy` container
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+# ...
+spec:
+  result:
+    options:
+      deployments:
+        tekton-results-watcher:
+          spec:
+            template:
+              spec:
+                containers:
+                - name: kube-rbac-proxy
+                  args:
+                    - --secure-listen-address=0.0.0.0:8443
+                    - --upstream=http://127.0.0.1:9090/
+                    - --logtostderr=true
+                  image: registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.12
+# ...
+----
+
+* With this release, the high availability options of the {pipelines-shortname} controller are enhanced with `StatefulSet` ordinals as an alternative to the existing leader election mechanism. This enables the Operator to use either quick recovery with leader election or consistent workload distribution with `StatefulSet` ordinals.
++
+:FeatureName: Using `StatefulSet` ordinals for high availability
+include::snippets/technology-preview.adoc[]
++
+Leader election, which is the default option, offers failover capabilities but might cause hot-spotting. In contrast, the new `StatefulSet` ordinals approach ensures keys are evenly spread across replicas for a more balanced workload distribution.
++
+You can switch to the `StatefulSet` ordinals method by setting the `statefulset-ordinals` parameter to `true` in the `TektonConfig` custom resource:
++
+.Example of enabling the StatefulSet ordinals feature
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+# ...
+  pipeline:
+    performance:
+      disable-ha: false
+      buckets: 4
+      replicas: 4
+      statefulset-ordinals: true
+# ...
+----
+
+[id="triggers-new-features-1-18_{context}"]
+=== Triggers
+
+* With this release, the `EventListener` object for {pipelines-shortname} triggers now includes the `ImagePullSecrets` field to specify a secret that the listener uses to pull images from private registries.
++
+.Example of using the ImagePullSecrets field
```

Comment:
```suggestion
.Example of using the `ImagePullSecrets` field
```
added the backticks here and the other example title for consistency with the text that has them in backticks.